pyproject.toml

```diff
@@ -1,12 +1,12 @@
-[build-system]
-requires = ["setuptools>=61.0"]
-build-backend = "setuptools.build_meta"
-
-[project]
-authors = [
-    {name = "Saritasa", email = "pypi@saritasa.com"},
-]
-requires-python = ">=3.9"
+[tool.poetry]
```

Comment:
Please add `maintainers`, `homepage`, `repository`

Reply:
We actually don't have homepage, but repository link might be useful. I'll add `documentation` also

Reply:
What about docs page?

Reply:
https://django-import-export-extensions.readthedocs.io/en/latest/